microservices/Keli-Katheya-User-Registration/src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+server.port=0
+spring.application.name=User-Registration
+
+token.expiration_time=8400000
+token.secret=abcnvain12345
+authenticate.url=/users/login
```

Comment:
This need not be configurable, a final constant string within code should be fine

Reply:
moved to constant files